src/packages/solid/components/Checkbox/Checkbox.styles.ts

```diff
@@ -46,8 +27,8 @@ const size = theme.spacer.xxl;
 
```

Comment:
this isn't related to this PR, but let's the `strokeWidth` and `size` consts here and just reference the theme so we keep the file uniform